### DISCUSSION TITLE
StepFunctions でエラーが発生したらアラームを出してほしい

### DISCUSSION THREAD

**Discussion by koichimurakami** [mbc-net/mbc-cqrs-serverless#96] [Ideas]
StepFunctionsで処理が失敗した際にはアラームを出すようにしてほしいです。
例えばアラーム用のSNSを作成し、StepFunctionsがエラーを検知したらSNSに通知するようにしてほしいです。

**Comment by baquan1708** [comment]
@koichimurakami 

There have already 2 SNS topic in our infrastructure.
- Main SNS: contain event about task status and command status
- Alarm SNS: contain alarm event

So, I think we can use exist SNS topic, alarm SNS, for raising an alarm when a Step Functions process fails. Would you agree with this approach?

**Reply by koichimurakami** [reply]
@baquan1708
Thank you. 
I agree.

**Reply by baquan1708** [reply]
@koichimurakami cc: @Tuan-Anh-Nguyen-pionero 

I've just create https://github.com/mbc-net/mbc-cqrs-serverless/pull/100 for this feature. Could you please review it?
